_posts/2016-02-14-version-0-dot-6-now-available.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: "Version 0.6 Now Available"
+date: 2016-02-14 05:22:23 +0530
+categories: [news, release]
+author: hashhar
+download: true
+---
+# Open Live Writer v0.6.0.0 is now available!
+
+After about two months since the initial public release the team and community have managed to add features, fix issues and fulfill the demand for some of the more requested features like spell checking (it will only work on Windows 8 and later though) and support for Blogger categories. We hope you enjoy the new features as much as we enjoyed working on them. Read on to learn more about what issues have been fixed and what features have been added.
+
+The next time you start Open Live Writer and are connected to the internet, it should get updated automatically. But if you can't wait to try out the goodness, download the [updated setup here](https://openlivewriter.azureedge.net/stable/Releases/OpenLiveWriterSetup.exe).
+
+## Release Notes
+
+For spell checking to work you'll need to have Windows 8 or newer installed (Windows 7 does not have spell checking support) and you’ll need to install your preferred input language in Windows to enable spell checking in that language. Only one language can be spell checked in Open Live Writer at a time, but you can change the language at any time by going to **File > Options > Spelling** and choosing a new dictionary language.
```

Comment:
Can we get a link to the instructions on how to install the input language. There should be a link on the release notes. We should definitely have this mentioned in here too.